docs/src/snap/howto/epa.md

```diff
@@ -0,0 +1,1061 @@
+
+```
+sudo k8s kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/sriov-network-device-plugin/master/deployments/sriovdp-daemonset.yaml
+```
+
+#### SRIOV CNI 
+
+```
+sudo k8s kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/sriov-cni/master/images/sriov-cni-daemonset.yaml
+```
+
+#### Multus NetworkAttachmentDefinition 
+
+Create the sriov-nad.yaml NetworkAttachmentDefinition
+
+```
+cat <<EOF | tee sriov-nad.yaml
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+ name: sriov-net1
+ annotations:
+   k8s.v1.cni.cncf.io/resourceName: intel.com/intel_sriov_netdevice
+spec:
+ config: '{
+ "type": "sriov",
+ "cniVersion": "0.3.1",
+ "name": "sriov-network",
+ "ipam": {
+   "type": "host-local",
+   "subnet": "10.18.2.153/24", # The subnet for SR-IOV VFs
+   "routes": [{
+     "dst": "0.0.0.0/0"
+   }],
+   "gateway": "10.18.2.1" 
+ }
+}'
+EOF
+
+cat <<EOF | tee dpdk-nad.yaml
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+ name: dpdk-net1
+ annotations:
+   k8s.v1.cni.cncf.io/resourceName: intel.com/intel_sriov_dpdk
+spec:
+ config: '{
+ "type": "sriov",
+ "cniVersion": "0.3.1",
+ "name": "dpdk-network"
+}'
+EOF
+```
+
+Apply the two NetworkAttachmentDefinition definition files
+
+```
+sudo k8s kubectl create -f ./sriov-nad.yaml
+sudo k8s kubectl create -f ./dpdk-nad.yaml
+```
+
+## Testing
+
+It is important to verify that all of these enabled features are working as
+expected before relying on them. This section deals with how to verify
+everything is working as expected.
+
+### Testing HugePages 
+
+Verify that HugePages are allocated on your Kubernetes nodes. You can do this
+by checking the node's capacity and allocatable resources:
+
+```
+sudo k8s kubectl get nodes
+```
+
+```
+```
+NAME          STATUS   ROLES                  AGE   VERSION
+pc6b-rb4-n1   Ready    control-plane,worker   22h   v1.30.2
+pc6b-rb4-n3   Ready    worker                 22h   v1.30.2
+```
+
+
+```
+  hugepages-1Gi:      1000Gi
+  hugepages-2Mi:      0
+  hugepages-1Gi:      1000Gi
+  hugepages-2Mi:      0
+  hugepages-1Gi      0 (0%)      0 (0%)
+  hugepages-2Mi      0 (0%)      0 (0%)
+```
+
+So we have 1000 huge pages of 1Gi size each and we have a worker node labelled
+properly. Then you can create a Pod that explicitly requests one 1G Huge Page
+in its resource limits:
+
+```
+cat <<EOF | sudo k8s kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: hugepage-test-ubuntu
+spec:
+  containers:
+  - name: ubuntu-container
+    image: ubuntu:latest
+    command: ["sleep", "infinity"]
+    resources:
+      limits:
+        hugepages-1Gi: 1Gi
+        memory: 128Mi
+      requests:
+        memory: 128Mi
+EOF
+```
+
+Note: To ensure proper resource management and prevent conflicts, Kubernetes
+enforces that a pod requesting HugePages also explicitly requests a minimum
+amount of either CPU or standard memory resources.
```

Comment:
```suggestion
```{note}
To ensure proper resource management and prevent conflicts, Kubernetes
enforces that a pod requesting HugePages also explicitly requests a minimum
amount of either CPU or standard memory resources.
```
Add closing tics please.